bitnami/redis/README.md

```diff
@@ -85,6 +85,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `kubeVersion`             | Override Kubernetes version                                                                                    | `""`            |
 | `nameOverride`            | String to partially override common.names.fullname                                                             | `""`            |
 | `fullnameOverride`        | String to fully override common.names.fullname                                                                 | `""`            |
+| `namespaceOverride`        | String to fully override common.names.namespace                                                                 | `""`            |
```

Comment:
`readme-generator` workflow is [failing](https://github.com/bitnami/charts/actions/runs/7086864328/job/19285943612?pr=21365) because you protected your PR and branch to avoid changes from our side. There is no problem about that, but then you need to rung [readme-generator](https://github.com/bitnami/readme-generator-for-helm) locally to ensure there is no unexpected changes. You can install directly the [node package](https://www.npmjs.com/package/@bitnami/readme-generator-for-helm)

Reply:
All good with the readme 😄 

![image](https://github.com/bitnami/charts/assets/38658722/98191620-c6e7-4f6b-ae2d-c3dc3f5c32ee)

Reply:
Don't know why the generator didn't make the change on the first try. Now it's fixed :)